components/dashboard/src/user-context.tsx

```diff
@@ -26,6 +26,25 @@ const UserContextProvider: React.FC = ({ children }) => {
                 client.clear();
             }
             setUser(updatedUser);
+
+            // Schedule a periodic refresh of JWT cookie
+            const w = window as any;
+            const _gp = w._gp || (w._gp = {});
+
+            const frequencyMs = 1000 * 60 * 60; // 1 hour
+            if (!_gp.jwttimer) {
+                // Store the timer on the window, to avoid queuing up multiple
+                _gp.jwtTimer = setInterval(() => {
+                    fetch("/api/auth/jwt-cookie", {
```

Comment:
The endpoint itself is behind a feature flag, so we don't necessarily need to feature flag here. But if we wanted to, would it be problematic to depend on the FeatureFlagContext? It could create a cyclic dependency as FeatureFlagContext depends on the User.